### DISCUSSION TITLE
Support for multiple charts with headings

### DISCUSSION THREAD

**Discussion by andersennl** [tone-row/flowchart-fun#53] [Features / Syntax]
It would be useful to be able to render multiple charts, separated by blank lines, each with its own heading.

Example:
```
_Scenario 1
Service 1
 connection: (3)
Service 2
 connection: (2)

_Scenario 2
Service 1
 connection: (10)
Service3
 connection: (8)
```

**Comment by rob-gordon** [comment]
@andersennl Moved this over to a discussion so that other people can express interests and suggest syntax. Thanks for your suggestion!

**Reply by andersennl** [reply]
The syntax in my example was just picked totally random.
